contrib/completion/fish/docker.fish

```diff
@@ -485,13 +573,32 @@ complete -c docker -A -f -n '__fish_seen_subcommand_from save' -l help -d 'Print
 complete -c docker -A -f -n '__fish_seen_subcommand_from save' -s o -l output -d 'Write to an file, instead of STDOUT'
 complete -c docker -A -f -n '__fish_seen_subcommand_from save' -a '(__fish_print_docker_images)' -d "Image"
```

Comment:
I just noticed that `-A` is deprecated and no longer does something (according to `complete --help`);

```
       -A and --authoritative no longer do anything and are silently ignored.
```


While you're working on this, perhaps you can remove the use of `-A` in this file?